yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/ShareSessionReceipt.java

```diff
@@ -0,0 +1,208 @@
+package com.yoti.api.client.spi.remote.call;
+
+import java.time.OffsetDateTime;
+import java.util.Base64;
+import java.util.Objects;
+import java.util.Optional;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(builder = ShareSessionReceipt.Builder.class)
+public final class ShareSessionReceipt {
```

Comment:
I called it `ShareSessionReceipt` to differenciate it from the `Receipt` v1. I guess I can call both the same since there is no cross reference between them tho.